src/entities/inventory.entity.js

```diff
@@ -0,0 +1,22 @@
+export default class Inventory {
+  //intenger
+  #id;
```

Comment:
Shouldn't this be public instead? I guess you would like to set and read this variable whenever you want.